lib/tasks/matches.rake

```diff
@@ -0,0 +1,13 @@
+namespace :matches do
+  desc "Send follow up mails"
+  task follow_up_mails: :environment do
+    Match.active
+         .where("created_at < ?", Rails.configuration.follow_up_matches_after.from_now)
+         .where(follow_up_mail_sent_at: nil)
+         .each do |match|
```

Comment:
Couldn't think of a good name for a scope here. So far this is the only place we use this query but maybe there will be others eventually?